src/main.c

```diff
@@ -2416,13 +2424,17 @@ void on_key(GLFWwindow *window, int key, int scancode, int action, int mods) {
                 g->isDeleting = false;
 			}
 		}
+        //
         // moves the sun roughly 3 hours forward when key is pressed
+        //
         if (key == CRAFT_KEY_TIMETRAVEL) {
             glfwSetTime(glfwGetTime() + timeMachine);
 		}
-        // moves the sun roughly 3 hours backwards when key is pressed
+        //
+        // moves the sun roughly 3 hours backwards when key is pressed - cannot go past time of creation
+        //
         if (key == CRAFT_KEY_TIMEUNTRAVEL) {
-            glfwSetTime(glfwGetTime() - timeMachine); // cannot go past time of creation
+            glfwSetTime(glfwGetTime() - timeMachine); 
```

Comment:
It's totally fine to still comment individual lines of code if you want to explain something about that particular line.  Don't think you have to include everything in the doxygen comment.